### PR TITLE
Fix typos in docs

### PR DESCRIPTION
**Pull request by saintmalik**
## What this pr does

Made corrections to the typos i found in the documentations.

### CODE REVIEW

File: v20.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md [hunk 1]
```diff
@@ -280,7 +280,7 @@ If you want to run on another cloud or on-premises, use this [basic network test
 
 ### Set up VPC peering
 
-For pods to communciate across three separate Kubernetes clusters, the VPCs in all regions need to be peered. Network traffic can then be routed between the VPCs. For more information about VPC peering, see the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html).
```
Comment (saintmalik) [inline v20.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md:283]: communciate >> communicate

File: _includes/sidebar.js.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
       baseUrl: "{{ site.baseurl }}",
       isVersionDirectory: function (d) {
         // Version directories either are one of the named version
-        // aliases specifed in _config.yml or have the form "vX.X",
+        // aliases specified in _config.yml or have the form "vX.X",
```
Comment (saintmalik) [inline _includes/sidebar.js.html:9]: specifed >> specified

File: v21.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md [hunk 3]
```diff
@@ -305,7 +305,7 @@ For each region, navigate to the Security Groups section of the [Amazon EC2 cons
 Remember to create these inbound rules in all 3 regions. This enables CockroachDB to communicate across each Kubernetes cluster.
 {{site.data.alerts.end}}
 
-#### Inter-region communciation
```
Comment (saintmalik) [inline v21.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md:308]: communciation >> communication

File: v21.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md [hunk 4]
```diff
@@ -280,7 +280,7 @@ If you want to run on another cloud or on-premises, use this [basic network test
 
 ### Set up VPC peering
 
-For pods to communciate across three separate Kubernetes clusters, the VPCs in all regions need to be peered. Network traffic can then be routed between the VPCs. For more information about VPC peering, see the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html).
```
Comment (saintmalik) [inline v21.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md:283]: communciate >> communicate

File: v21.2/auto-tls.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ With Auto TLS, your cluster creates the CA (certificate authority) certificate a
 - Additional CLI instructions and feedback.
 {{site.data.alerts.end}}
 
-Auto TLS simplifies the default method for [creating secure clusters](secure-a-cluster.html). Rather than manually generating certificates and keys and distributing them to all nodes, you run a one-time `cockroach connect` command for each node, specifiying a common token. A CA cert and key are distributed to all nodes, along with all other certificates that nodes require to communicate with other nodes and with clients, all signed by the CA.
```
Comment (saintmalik) [inline v21.2/auto-tls.md:24]: specifiying >>  specifying

File: v21.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md [hunk 6]
```diff
@@ -305,7 +305,7 @@ For each region, navigate to the Security Groups section of the [Amazon EC2 cons
 Remember to create these inbound rules in all 3 regions. This enables CockroachDB to communicate across each Kubernetes cluster.
 {{site.data.alerts.end}}
 
-#### Inter-region communciation
```
Comment (saintmalik) [inline v21.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md:308]: communciation >> communication

File: v21.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md [hunk 7]
```diff
@@ -280,7 +280,7 @@ If you want to run on another cloud or on-premises, use this [basic network test
 
 ### Set up VPC peering
 
-For pods to communciate across three separate Kubernetes clusters, the VPCs in all regions need to be peered. Network traffic can then be routed between the VPCs. For more information about VPC peering, see the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html).
```
Comment (saintmalik) [inline v21.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md:283]: communciate >> communicate

File: v21.1/auto-tls.md [hunk 8]
```diff
@@ -21,7 +21,7 @@ With Auto TLS, your cluster creates the CA (certificate authority) certificate a
 - Additional CLI instructions and feedback.
 {{site.data.alerts.end}}
 
-Auto TLS simplifies the default method for [creating secure clusters](secure-a-cluster.html). Rather than manually generating certificates and keys and distributing them to all nodes, you run a one-time `cockroach connect` command for each node, specifiying a common token. A CA cert and key are distributed to all nodes, along with all other certificates that nodes require to communicate with other nodes and with clients, all signed by the CA.
```
Comment (saintmalik) [inline v21.1/auto-tls.md:24]: specifiying >> specifying

File: v20.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md [hunk 9]
```diff
@@ -305,7 +305,7 @@ For each region, navigate to the Security Groups section of the [Amazon EC2 cons
 Remember to create these inbound rules in all 3 regions. This enables CockroachDB to communicate across each Kubernetes cluster.
 {{site.data.alerts.end}}
 
-#### Inter-region communciation
```
Comment (saintmalik) [inline v20.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md:308]: communciation >> communication

File: v20.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md [hunk 10]
```diff
@@ -280,7 +280,7 @@ If you want to run on another cloud or on-premises, use this [basic network test
 
 ### Set up VPC peering
 
-For pods to communciate across three separate Kubernetes clusters, the VPCs in all regions need to be peered. Network traffic can then be routed between the VPCs. For more information about VPC peering, see the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html).
```
Comment (saintmalik) [inline v20.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md:283]: communciate >> communicate

File: v20.2/authorization.md [hunk 11]
```diff
@@ -24,7 +24,7 @@ Throughout the documentation, however, we refer to a "user" or "role" based on t
 
 ## Users
 
-A SQL user (a role with `LOGIN` priviliges) can interact with a CockroachDB database using the [built-in SQL shell](cockroach-sql.html) or through an application.
```
Comment (saintmalik) [inline v20.2/authorization.md:27]: priviliges >> privileges

File: v20.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md [hunk 12]
```diff
@@ -305,7 +305,7 @@ For each region, navigate to the Security Groups section of the [Amazon EC2 cons
 Remember to create these inbound rules in all 3 regions. This enables CockroachDB to communicate across each Kubernetes cluster.
 {{site.data.alerts.end}}
 
-#### Inter-region communciation
```
Comment (saintmalik) [inline v20.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md:308]: communciation >> communication

**Comment by saintmalik** [comment]
@ericharmeling @ianjevans can you look into this please

**Review by ianjevans — APPROVED** [review]

**Pull request merged**